ultralytics/nn/autobackend.py

```diff
@@ -201,6 +201,10 @@ def __init__(
                 cuda = False
             LOGGER.info(f"Preferring ONNX Runtime {providers[0]}")
             if onnx:
+                check_requirements("onnxslim")
+                import onnxslim
+
+                w = onnxslim.slim(w, dtype="fp32").SerializeToString()
```

Comment:
@Y-T-G Do you think there will be a situation where users want to use half precision for inference, or should we add an option for them.